Section/pipeline (#1)
MrLongbottom/P9-Rapport: sections/Pipeline.tex

\section{Pipeline}

\input{figures/pipeline_figure.tex}
The pipeline of our framework is divided into five phases which are displayed in \autoref{fig:pipeline}.

\subsection{Article dataset}
We have a dataset consisting of articles from the media group Nordjyske. Their primary focus is to maintain a variety of local newspapers within the North Jutland region of Denmark. 
The data ranges from 2017-2019, where a total of $\sim$270.000 articles have been extracted from their database.

\subsection{Preprocessing Phase}
The extracted articles use the format:
\begin{itemize}

> 	\item Id

Author: ```suggestion
	\item id
```

> 	\item Headline
> 	\item Body

Author: ```suggestion
	\item headline
	\item body
```

> 	\item Id
> 	\item Headline
> 	\item Body

Author: ```suggestion
	\item id
	\item headline
	\item body
```

Reviewer: Vi skal passe på med hvordan det ser ud her, og i andre afsnit der stadig bliver lavet.

Author: enig

> \section{Pipeline}

Reviewer: Slet "figures/pipeline.tex" hvis den ikke bruges.

\input{figures/pipeline_figure.tex}
The pipeline of our framework is divided into five phases which are displayed in \autoref{fig:pipeline}.

\subsection{Article dataset}
We have an data set consisting of articles from the media group Nordjyske. Their primary focus is to maintain a variety of local newspapers within the North Jutland region of Denmark. 
The data ranges from 2017-2019, where a total of ~270.000 articles have been extracted from their database.

> \subsection{Preprocessing}

Author: ```suggestion
\subsection{Preprocessing Phase}
```

The extracted articles use the format:
\begin{itemize}
	\item Id
	\item Headline
	\item Body
\end{itemize}

> When training the LDA model, we concatenate the Headline and Body for simplicity.

Author: ```suggestion
When training the LDA model, we concatenate the \emph{headline} and \emph{body} for simplicity.
```

The preprocessing phase is described further in detail in \todo[inline]{ref til preprocessing}.

> This phase applies some processes to simplify the dataset and remove redundant information. 

Reviewer 2: ```suggestion
This phase applies different \gls{NLP} methods to simplify the dataset and remove redundant information.
```

After the preprocessing phase, we are left with ~130.000 articles.

\subsection{LDA}
We apply the \acrfull{LDA} to the dataset to generate topic clusters based on the content of the articles. 
We describe the investigation and selection of hyper parameters in \todo{ref til LDA section}. 
After the model has been trained, we have a document-topic distribution matrix $\theta$ and a topic-word distribution matrix $\varphi$.

> $\theta$ will be used as clusters in the next phase.

Author: så vidt jeg husker er symbolerne fra artiklen. Ellers check dem på wikipedia.

> $\theta$ will be used as clusters in the next phase.

Reviewer: Ok, har tjekket begge steder. Af hvad jeg kan se, definerer wiki phi til at være word distributionen, mens artiklen bruger beta som distribution. Jeg vil nok bruge beta da det er fra Blei's egen artikel, men hvis i har andre meninger, så siger i bare til.

> After the model has been trained, we have a document-topic distribution matrix $\theta$ and a topic-word distribution matrix $\varphi$.
> $\theta$ will be used as clusters in the next phase.

Author: ```suggestion
After the model has been trained, we have a document-topic distribution matrix $\theta$ and a topic-word distribution matrix $\beta$.
$\theta$ will be used as clusters in the next phase.
```

\subsection{Clustering PageRank}
In this phase, we apply a clustering random walk\todo{cite clusteirng pagerank} to the articles and clusters given in the \gls{LDA} phase.
The algorithm yields a prioritized list of articles based on the query provided.

> \subsection{Query Processing}

Reviewer: ```suggestion
\subsection{Query Preprocessing}
```
Det er vel rettere preprocessing. Husk figuren.
